impl/src/settings.rs

```diff
@@ -288,6 +288,7 @@ fn default_raze_settings_field_gen_workspace_prefix() -> String {
 }
 
 fn default_raze_settings_field_genmode() -> GenMode {
+  eprintln!("WARNING: The [raze] setting `genmode` should be explicitly set to either `\"Remote\"` or `\"Vendored\"`");
```

Comment:
Mention that this will be mandatory soon (that's the most important thing to convey).

Also, if you're open to it, i'd actually prefer a different implementation, which adds an "Unspecified" variant to GenMode, and maps that to Vendored in the implementation (for now), and prints this warning here: https://github.com/google/cargo-raze/blob/7cd1f6fee5aa3ca539106d5030281f3a4e950b1e/impl/src/bin/cargo-raze.rs#L72-L73

I don't feel strongly about this and I think there are good arguments against dealing with the Unspecified variant everywher.

Reply:
Oh, actually, here is better: https://github.com/google/cargo-raze/blob/852a983b3956941e4ae1ceb2fbe4a858ad931c9f/impl/src/settings.rs#L330-L350

Reply:
Yeah, I can take care of this later today.

Reply:
Done